includes/formatting.php

```diff
@@ -66,6 +66,10 @@ function get_body_contents( $content ) {
  * @return int The number of replacements made.
  */
 function sideload_images( $post_id ) {
+	require_once ABSPATH . 'wp-admin/includes/media.php';
+	require_once ABSPATH . 'wp-admin/includes/file.php';
+	require_once ABSPATH . 'wp-admin/includes/image.php';
+
 	$post    = get_post( $post_id );
 	$content = $post->post_content;
 	$pattern = '/["\'](https?:\/\/images.airstory.co\/[^"\']+)/i';
```

Comment:
I'd consider using DOMDocument along with some extra checks for image data (I built a plugin for pulling in external image from post content) but it's not a blocker at all.